modules/api-svc/test/unit/api/utils.js

```diff
@@ -12,23 +12,28 @@ const Cache = require('~/lib/cache');
 
 /**
  * Get OpenAPI spec and Validator for specified server
- * @param serverType String
+ * @param {('OUTBOUND'|'INBOUND')} serverType
  * @return {Promise<{apiSpecs: Object, validator: Validator}>}
  */
 const readApiInfo = async (serverType) => {
-    const specPath = path.join(__dirname, `../../../src/${serverType}/api.yaml`);
+    let specPath;
+    if (serverType === 'OUTBOUND') {
```

Comment:
Please define enums for INBOUND/OUTBOUND.

Why change it everywhere when you can change it one place \:D/